l10nzh.properties

```diff
@@ -2,15 +2,23 @@ category=Localization
 description=SonarQube Chinese Pack
 homepageUrl=https://github.com/SonarCommunity/sonar-l10n-zh
 archivedVersions=10.4
-publicVersions=9.9,10.0,10.1,10.2,10.3,10.4.1,10.5,10.6,10.7
+publicVersions=9.9,10.0,10.1,10.2,10.3,10.4.1,10.5,10.6,10.7,24.12
 
 defaults.mavenGroupId=org.codehaus.sonar-plugins.l10n
 defaults.mavenArtifactId=sonar-l10n-zh-plugin
 
+24.12.description=Support SonarQube 24.12
+24.12.sqs=[10.8,LATEST]
+24.12.sqcb=[24.12,LATEST]
+24.12.sqVersions=[24.12,LATEST]
```

Comment:
This property is now obsolete. No need to include it in the block going forward since this version's compatibility starts after the split into SonarQube Server and SonarQube Community Build.